lua/autorun/client/cfc_attention_monitoring_cl_init.lua

```diff
@@ -61,24 +69,24 @@ local function formatAfkTime( rawTime )
 end
 
 local function drawIcon( ply )
-    if not IsValid( ply ) then
+    if not ply or ply == NULL then
```

Comment:
not all NULL's are equal, this should an IsValid, can be :IsValid() likely

Reply:
Could you be a bit more specific? From all my knowledge and tests, it does not matter in this context if I check IsValid or if it is NULL/nil or not

Reply:
It's undefined behavior that shouldn't be relied upon